composer.json

```diff
@@ -25,7 +28,7 @@
 	},
 	"type"       : "phpcodesniffer-standard",
 	"scripts"    : {
-		"post-install-cmd": "\"vendor/bin/phpcs\" --config-set installed_paths ../../..",
-		"post-update-cmd" : "\"vendor/bin/phpcs\" --config-set installed_paths ../../.."
+		"post-install-cmd": "\"vendor/bin/phpcs\" --config-set installed_paths ../../..,../../phpcompatibility/php-compatibility",
+		"post-update-cmd" : "\"vendor/bin/phpcs\" --config-set installed_paths ../../..,../../phpcompatibility/php-compatibility"
```

Comment:
Let's keep this code dry:

```json
"scripts": {
	"install-codestandards": "\"vendor/bin/phpcs\" --config-set installed_paths ../../..,../../phpcompatibility/php-compatibility",
	"post-install-cmd": "@install-codestandards",
	"post-update-cmd": "@install-codestandards"
}
```

Reply:
https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1433

Reply:
I'd be fine with those changes, but they can be added any time.
They don't necessarily need to be included in this release, especially as it is only relevant for WPCS development anyway.